<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tableCreation</class>
 <widget class="QDialog" name="tableCreation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="inputTbName"/>
     </item>
     <item row="0" column="0" alignment="Qt::AlignLeft">
      <widget class="QLabel" name="label_6">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>55</height>
        </size>
       </property>
       <property name="text">
        <string>Table Name:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QCheckBox" name="null_5">
       <property name="text">
        <string>Allow null?</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QCheckBox" name="priKey_3">
       <property name="text">
        <string>Primary key?</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QCheckBox" name="priKey_2">
       <property name="text">
        <string>Primary key?</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QCheckBox" name="null_2">
       <property name="text">
        <string>Allow null?</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QComboBox" name="variableType3"/>
     </item>
     <item row="1" column="4">
      <widget class="QCheckBox" name="null_1">
       <property name="text">
        <string>Allow null?</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QCheckBox" name="priKey_1">
       <property name="text">
        <string>Primary key?</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="variableType1"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Column Name #2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Column Name #4</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_8">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Column Name #7</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QComboBox" name="variableType7"/>
     </item>
     <item row="6" column="4">
      <widget class="QCheckBox" name="null_6">
       <property name="text">
        <string>Allow null?</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="inputCol7"/>
     </item>
     <item row="7" column="3">
      <widget class="QCheckBox" name="priKey_7">
       <property name="text">
        <string>Primary key?</string>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QCheckBox" name="null_7">
       <property name="text">
        <string>Allow null?</string>
       </property>
      </widget>
     </item>
     <item row="7" column="5">
      <widget class="QCheckBox" name="unique_7">
       <property name="text">
        <string>Unique?</string>
       </property>
      </widget>
     </item>
     <item row="6" column="5">
      <widget class="QCheckBox" name="unique_6">
       <property name="text">
        <string>Unique?</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QCheckBox" name="priKey_6">
       <property name="text">
        <string>Primary key?</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QComboBox" name="variableType6"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="inputCol1"/>
     </item>
     <item row="2" column="5">
      <widget class="QCheckBox" name="unique_2">
       <property name="text">
        <string>Unique?</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Column Name #6</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QCheckBox" name="unique_5">
       <property name="text">
        <string>Unique?</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QCheckBox" name="unique_4">
       <property name="text">
        <string>Unique?</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="inputCol6"/>
     </item>
     <item row="3" column="5">
      <widget class="QCheckBox" name="unique_3">
       <property name="text">
        <string>Unique?</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QCheckBox" name="unique_1">
       <property name="text">
        <string>Unique?</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="inputCol4"/>
     </item>
     <item row="4" column="2">
      <widget class="QComboBox" name="variableType4"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Column Name #3</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QCheckBox" name="null_4">
       <property name="text">
        <string>Allow null?</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QCheckBox" name="null_3">
       <property name="text">
        <string>Allow null?</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="inputCol2"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Column Name #1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="variableType2"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="inputCol3"/>
     </item>
     <item row="4" column="3">
      <widget class="QCheckBox" name="priKey_4">
       <property name="text">
        <string>Primary key?</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QComboBox" name="variableType5"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="inputCol5"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Column Name #5</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QCheckBox" name="priKey_5">
       <property name="text">
        <string>Primary key?</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_9">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Column Name #8</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="inputCol8"/>
     </item>
     <item row="8" column="2">
      <widget class="QComboBox" name="variableType8"/>
     </item>
     <item row="8" column="3">
      <widget class="QCheckBox" name="priKey_8">
       <property name="text">
        <string>Primary key?</string>
       </property>
      </widget>
     </item>
     <item row="8" column="4">
      <widget class="QCheckBox" name="null_8">
       <property name="text">
        <string>Allow null?</string>
       </property>
      </widget>
     </item>
     <item row="8" column="5">
      <widget class="QCheckBox" name="unique_8">
       <property name="text">
        <string>Unique?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QPushButton" name="acceptButton">
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Accept</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
